src/navier_stokes/INSVCStaggeredConservativeMassMomentumRKIntegrator.cpp

```diff
@@ -1106,29 +1124,30 @@ INSVCStaggeredConservativeMassMomentumIntegrator::INSVCStaggeredConservativeMass
     TBOX_ASSERT(d_V_composite_idx >= 0);
 #endif
 
-    const std::string rho_sc_name = "INSVCStaggeredConservativeMassMomentumIntegrator::RHO_SIDE_CENTERED";
-    d_rho_sc_var = var_db->getVariable(rho_sc_name);
-    if (d_rho_sc_var)
+    const std::string rho_sc_name = "INSVCStaggeredConservativeMassMomentumRKIntegrator::RHO_SIDE_CENTERED";
```

Comment:
Should these variables use the `d_object_name` to set up the variable name? Otherwise you can only allocate one of these objects at a time.

Reply:
(Same comment applies to the other variable names --- I will not mark those.)